rewrite-gradle/src/test/java/org/openrewrite/gradle/trait/GradleDependencyTest.java

```diff
@@ -69,9 +71,102 @@ void literal() {
         );
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {
+      "api",
+      "implementation",
+      "compileOnly",
+      "runtimeOnly",
+      "testImplementation",
+      "testCompileOnly",
+      "testRuntimeOnly",
+      "debugImplementation",
+      "releaseImplementation",
+      "androidTestImplementation",
+      "featureImplementation",
+      "annotationProcessor",
+      "kapt",
+      "ksp"
```

Comment:
These are not Gradle native configurations and instead come from a custom configuration that come from a plugin not defined within this test case which explains why they fail.